packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -639,7 +677,7 @@ describe.skip('component testing migration - defaults', () => {
   })
 })
 
-describe('Migration', { viewportWidth: 1200 }, () => {
+describe('Migration', { viewportWidth: 1200, retries: { openMode: 2, runMode: 2 } }, () => {
```

Comment:
You could just do `retries: 2` here, and not need to specify the values for each mode

Reply:
@ImCesar can you make this change? Thanks!